gr-audio/lib/portaudio/portaudio_sink.cc

```diff
@@ -121,8 +122,7 @@ int portaudio_sink_callback(const void* inputBuffer,
         self->d_nunderuns++;
         ssize_t r = ::write(2, "aU", 2); // FIXME change to non-blocking call
```

Comment:
I think: either we log warnings from the callback about failures consistently to stderr or to the logger, or better even, we make that configurable. However, logging the one kind of problems to the logger and the other to stderr: bad idea.

Reply:
addressed in #3024